app/styles/ember-frost-table/_frost-fixed-table.scss

```diff
@@ -48,6 +48,22 @@
     .ps-container > .ps-scrollbar-y-rail {
       display: none;
     }
+
+    // sass-lint:disable nesting-depth
+    .frost-table-row {
+      &.is-selected {
```

Comment:
Are we using `is-selected` in the list for selection? It will be nice to use the same class name.